### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by rwblickhan**
# Pull Request

## Description

Give a brief description of your changes:
Just a small formatting change.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 ## Description
```
Comment (bobheadxi) [inline .github/PULL_REQUEST_TEMPLATE.md]: where the emojis 😢

Comment (rwblickhan) [reply .github/PULL_REQUEST_TEMPLATE.md]: There's never been emojis in ours...

Comment (bobheadxi) [reply .github/PULL_REQUEST_TEMPLATE.md]: its proven that emojis in PR template improve developer productivity

Comment (rwblickhan) [reply .github/PULL_REQUEST_TEMPLATE.md]: Citation?

Comment (bobheadxi) [reply .github/PULL_REQUEST_TEMPLATE.md]: (Lin et al, 2019)

Comment (rwblickhan) [reply .github/PULL_REQUEST_TEMPLATE.md]: Wow, you're published? Congrats man! I think we should _definitely_ announce this at the next LP meeting 😛

**Pull request merged**